llrt_core/src/module_loader/resolver.rs

```diff
@@ -411,7 +411,40 @@ fn node_modules_paths(start: &str) -> Vec<Box<str>> {
     dirs
 }
 
-// TODO LOAD_PACKAGE_IMPORTS(X, DIR)
+// LOAD_PACKAGE_IMPORTS(X, DIR)
+fn load_package_imports(ctx: &Ctx<'_>, x: &str, dir: &str) -> Result<Option<String>> {
+    trace!("|  load_package_imports(x, dir): ({}, {})", x, dir);
+    // 1. Find the closest package scope SCOPE to DIR.
+    let mut package_json_file: Vec<u8>;
+    let package_json: BorrowedValue;
+    match find_the_closest_package_scope(dir) {
+        // 2. If no scope was found, return.
+        None => {
+            return Ok(None);
+        },
+        Some(path) => {
+            package_json_file = fs::read(path.as_ref()).or_throw(ctx)?;
+            package_json = simd_json::to_borrowed_value(&mut package_json_file).or_throw(ctx)?;
+            // 3. If the SCOPE/package.json "imports" is null or undefined, return.
+            if !is_imports_field_exists(&package_json) {
+                return Ok(None);
+            }
```

Comment:
This check seems redundant as we check the same thing in `package_imports_resolve`. Please remove.